openpathsampling/experimental/simstore/storage.py

```diff
@@ -587,20 +585,33 @@ def __iter__(self):
 
     def __getitem__(self, item):
         len_self = len(self)
-        if not (-len_self <= item < len_self):
-            raise IndexError("table index out of range")
-
-        if item < 0:
-            item += len(self)
+        if item is slice:
```

Comment:
I'm utterly confused by this line. It seems to work (tests pass!), but I thought it would need to be:

```python
        if item isinstance(slice):
```

Any link to docs on why this works? I thought this would test against the builtin `slice`, and the item passed to `__getitem__` would be an instance of `slice`, not the class itself.

Reply:
It doesn't work :facepalm:, I guess this is why we have code review 

The test is actually not hitting this code... but instead does slicing on `PseudoTable._sequence`, which is a `list` by default, so of course it behaves as a list :P Let me ponder on how to actually test this and make sure we catch the real error xD ("tuple" also shows up if you try to slice a list with a tuple, as a good error message should do)